### ISSUE TITLE
Corrupt name object when parsing

### ISSUE THREAD

**Issue by simon-franz** [pdfcpu/pdfcpu#252]
Hi there, 
I've received a pdf produced by "Kdan PocketScanner" and want to merge it with a normal pdf. It seems there is an error in the pdf which prevents me from doing that.
Maybe someone else has the same problem.

I've run
`pdfcpu validate -v "file.pdf" 2> "error.log"`

` INFO: 2020/11/27 13:38:43 PDF Version 1.5 conforming reader
Fatal: pdfcpu: parse: corrupt name object
github.com/pdfcpu/pdfcpu/pkg/pdfcpu.init
	/Users/horstrutter/go/src/github.com/pdfcpu/pdfcpu/pkg/pdfcpu/parse.go:37
runtime.doInit
	/Users/horstrutter/gotip/src/runtime/proc.go:5625
runtime.doInit
	/Users/horstrutter/gotip/src/runtime/proc.go:5620
runtime.main
	/Users/horstrutter/gotip/src/runtime/proc.go:191
runtime.goexit
	/Users/horstrutter/gotip/src/runtime/asm_amd64.s:1374
dereferenceObject: problem dereferencing object 9
github.com/pdfcpu/pdfcpu/pkg/pdfcpu.dereferenceObject
	/Users/horstrutter/go/src/github.com/pdfcpu/pdfcpu/pkg/pdfcpu/read.go:2192
github.com/pdfcpu/pdfcpu/pkg/pdfcpu.dereferenceObjects
	/Users/horstrutter/go/src/github.com/pdfcpu/pdfcpu/pkg/pdfcpu/read.go:2292
github.com/pdfcpu/pdfcpu/pkg/pdfcpu.dereferenceXRefTable
	/Users/horstrutter/go/src/github.com/pdfcpu/pdfcpu/pkg/pdfcpu/read.go:2371
github.com/pdfcpu/pdfcpu/pkg/pdfcpu.Read
	/Users/horstrutter/go/src/github.com/pdfcpu/pdfcpu/pkg/pdfcpu/read.go:79
github.com/pdfcpu/pdfcpu/pkg/api.ReadContext
	/Users/horstrutter/go/src/github.com/pdfcpu/pdfcpu/pkg/api/api.go:48
github.com/pdfcpu/pdfcpu/pkg/api.Validate
	/Users/horstrutter/go/src/github.com/pdfcpu/pdfcpu/pkg/api/validate.go:42
github.com/pdfcpu/pdfcpu/pkg/api.ValidateFile
	/Users/horstrutter/go/src/github.com/pdfcpu/pdfcpu/pkg/api/validate.go:91
github.com/pdfcpu/pdfcpu/pkg/cli.Validate
	/Users/horstrutter/go/src/github.com/pdfcpu/pdfcpu/pkg/cli/cli.go:33
github.com/pdfcpu/pdfcpu/pkg/cli.Process
	/Users/horstrutter/go/src/github.com/pdfcpu/pdfcpu/pkg/cli/process.go:35
main.process
	/Users/horstrutter/go/src/github.com/pdfcpu/pdfcpu/cmd/pdfcpu/process.go:88
main.processValidateCommand
	/Users/horstrutter/go/src/github.com/pdfcpu/pdfcpu/cmd/pdfcpu/process.go:126
main.commandMap.process
	/Users/horstrutter/go/src/github.com/pdfcpu/pdfcpu/cmd/pdfcpu/cmd.go:145
main.main
	/Users/horstrutter/go/src/github.com/pdfcpu/pdfcpu/cmd/pdfcpu/main.go:55
runtime.main
	/Users/horstrutter/gotip/src/runtime/proc.go:204
runtime.goexit
	/Users/horstrutter/gotip/src/runtime/asm_amd64.s:1374
`

Is there anything else I can help with to maybe merge it with other pdfs with pdfcpu ?
Thanks in advance

**Comment by hhrutter** [comment]
Can you get me the output of `pdfcpu val -vv file.pdf ` for analysis.
Thank you for using pdfcpu 💚

**Comment by simon-franz** [comment]
Yeah, sure

` READ: 2020/11/27 14:42:43 Read: begin
 INFO: 2020/11/27 14:42:43 PDF Version 1.5 conforming reader
 READ: 2020/11/27 14:42:43 readXRefTable: begin
 READ: 2020/11/27 14:42:43 scanning for offsetLastXRefSection starting at 139195
 READ: 2020/11/27 14:42:43 Offset last xrefsection: 139367
 READ: 2020/11/27 14:42:43 buildXRefTableStartingAt: begin
 READ: 2020/11/27 14:42:43 headerVersion begin
 READ: 2020/11/27 14:42:43 headerVersion: end, found header version: 1.3
 READ: 2020/11/27 14:42:43 newPositionedReader: positioned to offset: 139367
 READ: 2020/11/27 14:42:43 line: <xref>
 READ: 2020/11/27 14:42:43 buildXRefTableStartingAt: found xref section
 READ: 2020/11/27 14:42:43 parseXRefSection begin
 READ: 2020/11/27 14:42:43 parseXRefSection: <0 13>
 READ: 2020/11/27 14:42:43 parseXRefTableSubSection: begin
 READ: 2020/11/27 14:42:43 detected xref subsection, startObj=0 length=13
 READ: 2020/11/27 14:42:43 parseXRefTableEntry: begin
 READ: 2020/11/27 14:42:43 parseXRefTableEntry: Object #0 is unused, next free is object#0, generation=65535
 READ: 2020/11/27 14:42:43 parseXRefTableEntry: Insert new xreftable entry for Object 0
 READ: 2020/11/27 14:42:43 parseXRefTableEntry: end
 READ: 2020/11/27 14:42:43 parseXRefTableEntry: begin
 READ: 2020/11/27 14:42:43 parseXRefTableEntry: Object #1 is in use at offset=15, generation=0
 READ: 2020/11/27 14:42:43 parseXRefTableEntry: Insert new xreftable entry for Object 1
 READ: 2020/11/27 14:42:43 parseXRefTableEntry: end
 READ: 2020/11/27 14:42:43 parseXRefTableEntry: begin
 READ: 2020/11/27 14:42:43 parseXRefTableEntry: Object #2 is in use at offset=103, generation=0
 READ: 2020/11/27 14:42:43 parseXRefTableEntry: Insert new xreftable entry for Object 2
 READ: 2020/11/27 14:42:43 parseXRefTableEntry: end
 READ: 2020/11/27 14:42:43 parseXRefTableEntry: begin
 READ: 2020/11/27 14:42:43 parseXRefTableEntry: Object #3 is in use at offset=162, generation=0
 READ: 2020/11/27 14:42:43 parseXRefTableEntry: Insert new xreftable entry for Object 3
 READ: 2020/11/27 14:42:43 parseXRefTableEntry: end
 READ: 2020/11/27 14:42:43 parseXRefTableEntry: begin
 READ: 2020/11/27 14:42:43 parseXRefTableEntry: Object #4 is in use at offset=326, generation=0
 READ: 2020/11/27 14:42:43 parseXRefTableEntry: Insert new xreftable entry for Object 4
 READ: 2020/11/27 14:42:43 parseXRefTableEntry: end
 READ: 2020/11/27 14:42:43 parseXRefTableEntry: begin
 READ: 2020/11/27 14:42:43 parseXRefTableEntry: Object #5 is in use at offset=396, generation=0
 READ: 2020/11/27 14:42:43 parseXRefTableEntry: Insert new xreftable entry for Object 5
 READ: 2020/11/27 14:42:43 parseXRefTableEntry: end
 READ: 2020/11/27 14:42:43 parseXRefTableEntry: begin
 READ: 2020/11/27 14:42:43 parseXRefTableEntry: Object #6 is in use at offset=498, generation=0
 READ: 2020/11/27 14:42:43 parseXRefTableEntry: Insert new xreftable entry for Object 6
 READ: 2020/11/27 14:42:43 parseXRefTableEntry: end
 READ: 2020/11/27 14:42:43 parseXRefTableEntry: begin
 READ: 2020/11/27 14:42:43 parseXRefTableEntry: Object #7 is in use at offset=688, generation=0
 READ: 2020/11/27 14:42:43 parseXRefTableEntry: Insert new xreftable entry for Object 7
 READ: 2020/11/27 14:42:43 parseXRefTableEntry: end
 READ: 2020/11/27 14:42:43 parseXRefTableEntry: begin
 READ: 2020/11/27 14:42:43 parseXRefTableEntry: Object #8 is in use at offset=797, generation=0
 READ: 2020/11/27 14:42:43 parseXRefTableEntry: Insert new xreftable entry for Object 8
 READ: 2020/11/27 14:42:43 parseXRefTableEntry: end
 READ: 2020/11/27 14:42:43 parseXRefTableEntry: begin
 READ: 2020/11/27 14:42:43 parseXRefTableEntry: Object #9 is in use at offset=815, generation=0
 READ: 2020/11/27 14:42:43 parseXRefTableEntry: Insert new xreftable entry for Object 9
 READ: 2020/11/27 14:42:43 parseXRefTableEntry: end
 READ: 2020/11/27 14:42:43 parseXRefTableEntry: begin
 READ: 2020/11/27 14:42:43 parseXRefTableEntry: Object #10 is in use at offset=886, generation=0
 READ: 2020/11/27 14:42:43 parseXRefTableEntry: Insert new xreftable entry for Object 10
 READ: 2020/11/27 14:42:43 parseXRefTableEntry: end
 READ: 2020/11/27 14:42:43 parseXRefTableEntry: begin
 READ: 2020/11/27 14:42:43 parseXRefTableEntry: Object #11 is in use at offset=929, generation=0
 READ: 2020/11/27 14:42:43 parseXRefTableEntry: Insert new xreftable entry for Object 11
 READ: 2020/11/27 14:42:43 parseXRefTableEntry: end
 READ: 2020/11/27 14:42:43 parseXRefTableEntry: begin
 READ: 2020/11/27 14:42:43 parseXRefTableEntry: Object #12 is in use at offset=139344, generation=0
 READ: 2020/11/27 14:42:43 parseXRefTableEntry: Insert new xreftable entry for Object 12
 READ: 2020/11/27 14:42:43 parseXRefTableEntry: end
 READ: 2020/11/27 14:42:43 parseXRefTableSubSection: end
 READ: 2020/11/27 14:42:43 parseXRefSection: All subsections read!
 READ: 2020/11/27 14:42:43 parseXRefSection: parsing trailer dict..
 READ: 2020/11/27 14:42:43 line (len 7) <trailer>
 READ: 2020/11/27 14:42:43 line: <>
 READ: 2020/11/27 14:42:43 line: <<<>
 READ: 2020/11/27 14:42:43 scanTrailer dictBuf after start tag: <<<>
 READ: 2020/11/27 14:42:43 scanTrailer dictBuf next line: </Root 1 0 R>
 READ: 2020/11/27 14:42:43 scanTrailer dictBuf next line: </Info 3 0 R>
 READ: 2020/11/27 14:42:43 scanTrailer dictBuf next line: </Size 13>
 READ: 2020/11/27 14:42:43 scanTrailer dictBuf next line: <>>>
 READ: 2020/11/27 14:42:43 processTrailer: trailerString: (len:39) <<< /Root 1 0 R /Info 3 0 R /Size 13 >> >
 READ: 2020/11/27 14:42:43 processTrailer: trailerDict:
<<
	<Info, (3 0 R)>
	<Root, (1 0 R)>
	<Size, 13>
>>
 READ: 2020/11/27 14:42:43 parseTrailerDict begin
 READ: 2020/11/27 14:42:43 parseTrailerInfo begin
 READ: 2020/11/27 14:42:43 parseTrailerInfo: Root object: (1 0 R)
 READ: 2020/11/27 14:42:43 parseTrailerInfo: Info object: (3 0 R)
 READ: 2020/11/27 14:42:43 parseTrailerInfo end
 READ: 2020/11/27 14:42:43 parseTrailerDict end
 READ: 2020/11/27 14:42:43 buildXRefTableStartingAt: end
TRACE: 2020/11/27 14:42:43 EnsureValidFreeList begin
TRACE: 2020/11/27 14:42:43 EnsureValidFreeList: empty free list.
 READ: 2020/11/27 14:42:43 readXRefTable: end
 READ: 2020/11/27 14:42:43 dereferenceXRefTable: begin
 READ: 2020/11/27 14:42:43 decodeObjectStreams: begin
 READ: 2020/11/27 14:42:43 decodeObjectStreams: end
 READ: 2020/11/27 14:42:43 dereferenceObjects: begin
 READ: 2020/11/27 14:42:43 dereferenceObject: begin, dereferencing object 0
 READ: 2020/11/27 14:42:43 free object 0
 READ: 2020/11/27 14:42:43 dereferenceObject: begin, dereferencing object 1
 READ: 2020/11/27 14:42:43 in use object 1
 READ: 2020/11/27 14:42:43 dereferenceObject: dereferencing object 1
 READ: 2020/11/27 14:42:43 ParseObject: begin, obj#1, offset:15
 READ: 2020/11/27 14:42:43 newPositionedReader: positioned to offset: 15
 READ: 2020/11/27 14:42:43 object: small obj w/o stream, parse until endobj
 READ: 2020/11/27 14:42:43 dict: end, #1
 READ: 2020/11/27 14:42:43 dereferenceObject: end obj 1 of 13
<<<
	<Outlines, (4 0 R)>
	<PageMode, UseOutlines>
	<Pages, (2 0 R)>
	<Type, Catalog>
>>>
 READ: 2020/11/27 14:42:43 logStream: no ObjectStreamDict
 READ: 2020/11/27 14:42:43 dereferenceObject: begin, dereferencing object 2
 READ: 2020/11/27 14:42:43 in use object 2
 READ: 2020/11/27 14:42:43 dereferenceObject: dereferencing object 2
 READ: 2020/11/27 14:42:43 ParseObject: begin, obj#2, offset:103
 READ: 2020/11/27 14:42:43 newPositionedReader: positioned to offset: 103
 READ: 2020/11/27 14:42:43 object: small obj w/o stream, parse until endobj
 READ: 2020/11/27 14:42:43 dict: end, #2
 READ: 2020/11/27 14:42:43 dereferenceObject: end obj 2 of 13
<<<
	<Count, 1>
	<Kids, [(6 0 R)]>
	<Type, Pages>
>>>
 READ: 2020/11/27 14:42:43 logStream: no ObjectStreamDict
 READ: 2020/11/27 14:42:43 dereferenceObject: begin, dereferencing object 3
 READ: 2020/11/27 14:42:43 in use object 3
 READ: 2020/11/27 14:42:43 dereferenceObject: dereferencing object 3
 READ: 2020/11/27 14:42:43 ParseObject: begin, obj#3, offset:162
 READ: 2020/11/27 14:42:43 newPositionedReader: positioned to offset: 162
 READ: 2020/11/27 14:42:43 object: small obj w/o stream, parse until endobj
 READ: 2020/11/27 14:42:43 dict: end, #3
 READ: 2020/11/27 14:42:43 dereferenceObject: end obj 3 of 13
<<<
	<Author, (Untitled)>
	<Creator, (iPad\040von\040marcus\040\0502\051)>
	<Producer, (Kdan\040PocketScanner)>
	<Title, (iPad\040von\040marcus\040\0502\051)>
>>>
 READ: 2020/11/27 14:42:43 logStream: no ObjectStreamDict
 READ: 2020/11/27 14:42:43 dereferenceObject: begin, dereferencing object 4
 READ: 2020/11/27 14:42:43 in use object 4
 READ: 2020/11/27 14:42:43 dereferenceObject: dereferencing object 4
 READ: 2020/11/27 14:42:43 ParseObject: begin, obj#4, offset:326
 READ: 2020/11/27 14:42:43 newPositionedReader: positioned to offset: 326
 READ: 2020/11/27 14:42:43 object: small obj w/o stream, parse until endobj
 READ: 2020/11/27 14:42:43 dict: end, #4
 READ: 2020/11/27 14:42:43 dereferenceObject: end obj 4 of 13
<<<
	<Count, 2>
	<First, (5 0 R)>
	<Last, (5 0 R)>
	<Type, Outline>
>>>
 READ: 2020/11/27 14:42:43 logStream: no ObjectStreamDict
 READ: 2020/11/27 14:42:43 dereferenceObject: begin, dereferencing object 5
 READ: 2020/11/27 14:42:43 in use object 5
 READ: 2020/11/27 14:42:43 dereferenceObject: dereferencing object 5
 READ: 2020/11/27 14:42:43 ParseObject: begin, obj#5, offset:396
 READ: 2020/11/27 14:42:43 newPositionedReader: positioned to offset: 396
 READ: 2020/11/27 14:42:43 object: small obj w/o stream, parse until endobj
 READ: 2020/11/27 14:42:43 dict: end, #5
 READ: 2020/11/27 14:42:43 dereferenceObject: end obj 5 of 13
<<<
	<Count, 1>
	<First, (9 0 R)>
	<Last, (9 0 R)>
	<Parent, (4 0 R)>
	<Title, (Untitled)>
	<Type, Outline>
>>>
 READ: 2020/11/27 14:42:43 logStream: no ObjectStreamDict
 READ: 2020/11/27 14:42:43 dereferenceObject: begin, dereferencing object 6
 READ: 2020/11/27 14:42:43 in use object 6
 READ: 2020/11/27 14:42:43 dereferenceObject: dereferencing object 6
 READ: 2020/11/27 14:42:43 ParseObject: begin, obj#6, offset:498
 READ: 2020/11/27 14:42:43 newPositionedReader: positioned to offset: 498
 READ: 2020/11/27 14:42:43 object: small obj w/o stream, parse until endobj
 READ: 2020/11/27 14:42:43 dict: end, #6
 READ: 2020/11/27 14:42:43 dereferenceObject: end obj 6 of 13
<<<
	<Contents, (7 0 R)>
	<MediaBox, [0 0 594.86 841.70]>
	<Parent, (2 0 R)>
	<Resources, <<
		<ProcSet, [PDF Text ImageB ImageC ImageI]>
		<XObject, <<
			<X1, (11 0 R)>
		>>>
	>>>
	<Type, Page>
>>>
 READ: 2020/11/27 14:42:43 logStream: no ObjectStreamDict
 READ: 2020/11/27 14:42:43 dereferenceObject: begin, dereferencing object 7
 READ: 2020/11/27 14:42:43 in use object 7
 READ: 2020/11/27 14:42:43 dereferenceObject: dereferencing object 7
 READ: 2020/11/27 14:42:43 ParseObject: begin, obj#7, offset:688
 READ: 2020/11/27 14:42:43 newPositionedReader: positioned to offset: 688
 READ: 2020/11/27 14:42:43 buffer: endInd=102 streamInd=28
 READ: 2020/11/27 14:42:43 object: small stream within buffer, parse until stream
 READ: 2020/11/27 14:42:43 pdfFilterPipeline: begin
 READ: 2020/11/27 14:42:43 streamDictForObject: end, Streamobject #7
 READ: 2020/11/27 14:42:43 LoadEncodedStreamContent: begin
<<
	<Length, (8 0 R)>
>>
 READ: 2020/11/27 14:42:43 int64Object begin: 8
 READ: 2020/11/27 14:42:43 dereferencedObject: dereferencing object 8
 READ: 2020/11/27 14:42:43 ParseObject: begin, obj#8, offset:797
 READ: 2020/11/27 14:42:43 newPositionedReader: positioned to offset: 797
 READ: 2020/11/27 14:42:43 object: small obj w/o stream, parse until endobj
 READ: 2020/11/27 14:42:43 int64Object end: 8
 READ: 2020/11/27 14:42:43 LoadEncodedStreamContent: new indirect streamLength:55
 READ: 2020/11/27 14:42:43 newPositionedReader: positioned to offset: 724
 READ: 2020/11/27 14:42:43 LoadEncodedStreamContent: seeked to offset:724
 READ: 2020/11/27 14:42:43 readContentStream: begin streamLength:55
 READ: 2020/11/27 14:42:43 readContentStream: count=55, buflen=55(37)
 READ: 2020/11/27 14:42:43 readContentStream: end
 READ: 2020/11/27 14:42:43 LoadEncodedStreamContent: end: len(streamDictRaw)=55
 READ: 2020/11/27 14:42:43 saveDecodedStreamContent: begin decode=false
 READ: 2020/11/27 14:42:43 dereferenceObject: end obj 7 of 13
<<<
	<Length, (8 0 R)>
>>>
 READ: 2020/11/27 14:42:43 logStream: no stream content
 READ: 2020/11/27 14:42:43 dereferenceObject: begin, dereferencing object 8
 READ: 2020/11/27 14:42:43 in use object 8
 READ: 2020/11/27 14:42:43 logStream: no ObjectStreamDict
 READ: 2020/11/27 14:42:43 handleCachedStreamDict: using cached object 8 of 13
<55>
 READ: 2020/11/27 14:42:43 dereferenceObject: begin, dereferencing object 9
 READ: 2020/11/27 14:42:43 in use object 9
 READ: 2020/11/27 14:42:43 dereferenceObject: dereferencing object 9
 READ: 2020/11/27 14:42:43 ParseObject: begin, obj#9, offset:815
 READ: 2020/11/27 14:42:43 newPositionedReader: positioned to offset: 815
 READ: 2020/11/27 14:42:43 object: small obj w/o stream, parse until endobj
Fatal: pdfcpu: parse: corrupt name object
github.com/pdfcpu/pdfcpu/pkg/pdfcpu.init
	/Users/horstrutter/go/src/github.com/pdfcpu/pdfcpu/pkg/pdfcpu/parse.go:37
runtime.doInit
	/Users/horstrutter/gotip/src/runtime/proc.go:5625
runtime.doInit
	/Users/horstrutter/gotip/src/runtime/proc.go:5620
runtime.main
	/Users/horstrutter/gotip/src/runtime/proc.go:191
runtime.goexit
	/Users/horstrutter/gotip/src/runtime/asm_amd64.s:1374
dereferenceObject: problem dereferencing object 9
github.com/pdfcpu/pdfcpu/pkg/pdfcpu.dereferenceObject
	/Users/horstrutter/go/src/github.com/pdfcpu/pdfcpu/pkg/pdfcpu/read.go:2192
github.com/pdfcpu/pdfcpu/pkg/pdfcpu.dereferenceObjects
	/Users/horstrutter/go/src/github.com/pdfcpu/pdfcpu/pkg/pdfcpu/read.go:2292
github.com/pdfcpu/pdfcpu/pkg/pdfcpu.dereferenceXRefTable
	/Users/horstrutter/go/src/github.com/pdfcpu/pdfcpu/pkg/pdfcpu/read.go:2371
github.com/pdfcpu/pdfcpu/pkg/pdfcpu.Read
	/Users/horstrutter/go/src/github.com/pdfcpu/pdfcpu/pkg/pdfcpu/read.go:79
github.com/pdfcpu/pdfcpu/pkg/api.ReadContext
	/Users/horstrutter/go/src/github.com/pdfcpu/pdfcpu/pkg/api/api.go:48
github.com/pdfcpu/pdfcpu/pkg/api.Validate
	/Users/horstrutter/go/src/github.com/pdfcpu/pdfcpu/pkg/api/validate.go:42
github.com/pdfcpu/pdfcpu/pkg/api.ValidateFile
	/Users/horstrutter/go/src/github.com/pdfcpu/pdfcpu/pkg/api/validate.go:91
github.com/pdfcpu/pdfcpu/pkg/cli.Validate
	/Users/horstrutter/go/src/github.com/pdfcpu/pdfcpu/pkg/cli/cli.go:33
github.com/pdfcpu/pdfcpu/pkg/cli.Process
	/Users/horstrutter/go/src/github.com/pdfcpu/pdfcpu/pkg/cli/process.go:35
main.process
	/Users/horstrutter/go/src/github.com/pdfcpu/pdfcpu/cmd/pdfcpu/process.go:88
main.processValidateCommand
	/Users/horstrutter/go/src/github.com/pdfcpu/pdfcpu/cmd/pdfcpu/process.go:126
main.commandMap.process
	/Users/horstrutter/go/src/github.com/pdfcpu/pdfcpu/cmd/pdfcpu/cmd.go:145
main.main
	/Users/horstrutter/go/src/github.com/pdfcpu/pdfcpu/cmd/pdfcpu/main.go:55
runtime.main
	/Users/horstrutter/gotip/src/runtime/proc.go:204
runtime.goexit
	/Users/horstrutter/gotip/src/runtime/asm_amd64.s:1374
`

**Comment by hhrutter** [comment]
Unfortunately I would need a sample file in order to fix this.
You can also send me an email hhrutter@gmail.com.
Thank you!

**Comment by simon-franz** [comment]
Thanks for your fast response!
I have contacted you with the corresponding file.

best regards

**Comment by hhrutter** [comment]
The PDFWriter used to create this file skips writing the StringLiteral expected as the value the outline title attribute. One can surely interpret an empty title here but nevertheless this screws up dict processing big time:

`9 0 obj <</Title 0x0A/Type /Outline 0x0A /Parent 5 0 R 0x0A /Dest 10 0 R 0x0A>> endobj`

Luckily we can provide a fix here that relies on a newline (0x0A) representing an eol (end of line) appearing after every key value pair of the dict but this is not something that's written in the PDF spec. To the contrary an end of line may appear anywhere within PDF objects as long as the imposed grammar is not violated.

Finally we end up with:
```
9:   offset=     815 generation=0 pdfcpu.Dict type=Outline
<<
	<Dest, (10 0 R)>
	<Parent, (5 0 R)>
	<Title, ()>
	<Type, Outline>
>>
```

.. and we still are not happy. `<Type, Outline>` is the next spec violation because it actually should be `<Type, Outlines>`:
![image](https://user-images.githubusercontent.com/11322155/100527953-acb1df80-31d7-11eb-8dca-b91583c18b14.png)

We extend relaxed validation and finally pdfcpu is able to digest this file.

**Comment by hhrutter** [comment]
Fixed with latest commit [cd73588](https://github.com/pdfcpu/pdfcpu/commit/cd735888c92861ba351e3d5d798f2e3bbb376365).
